app/controllers/folio/console/api/aasm_controller.rb

```diff
@@ -32,6 +32,19 @@ def event
             opts = {}
           end
 
+          if record.errors.any?
+            return render json: {
+                    data: cell("folio/console/state", record, opts).show,
+                    meta: {
+                      flash: {
+                        alert: t(".invalid_record_detail") +
+                        "<br>" +
+                        record.errors.full_messages.join("<br>")
```

Comment:
Nelibi se mi, ze tady lepis dohromady nejake HTML. Co pouzit pole `errors` jako v `render_failure` nize?

Reply:
A prosim `test/controllers/folio/console/api/aasm_controller_test.rb`

Reply:
nedari se mi zavolat https://github.com/sinfin/folio/blob/master/app/cells/folio/console/flash/flash.js#L73 tuhle metodu v api.js 
https://github.com/sinfin/folio/blob/master/app/assets/javascripts/folio/api.js#L132

zkusil jsem to takhle, ale response je undefined - zkusil jsem to dat do catch bloku ale tam se nepassuje cely response objekt ale jen response.error, nasmerujes me prosim?
```js
  return fetch(url, data).then(makeCheckResponse(false))
                         .then(responseToJson)
                         .then(window.Folio.Api.flashMessageFromMeta)
                         .then(window.FolioConsole.Flash.flashMessageFromApiErrors)
```

Reply:
akorat jeste nemuzu pustit test, pise ze mi chybi redactor

Reply:
> akorat jeste nemuzu pustit test, pise ze mi chybi redactor

Nakopiruj si soubory  z nejakeho projektu do test/dummy/vendor/assets/redactor/